python/ground/client.py

```diff
@@ -55,54 +58,67 @@ def _getRichVersionJson(self, reference, reference_parameters, tags, structure_v
         endpoint = "/" + item_type
         body = {"sourceKey": source_key, "name": name}
 
         if tags:
             body["tags"] = tags
 
-        return self._makePostRequest(endpoint, body)
+        return self._make_post_request(endpoint, body)
 
-    def _getItem(self, item_type, source_key):
-        return self._makeGetRequest("/" + item_type + "/" + source_key)
+    def _get_item(self, item_type, source_key):
+        return self._make_get_request("/" + item_type + "/" + source_key)
 
-    def _getItemLatestVersions(self, item_type, source_key):
-        return self._makeGetRequest("/" + item_type + "/" + source_key + "/latest")
+    def _get_item_latest_versions(self, item_type, source_key):
+        return self._make_get_request("/" + item_type + "/" + source_key + "/latest")
 
-    def _getItemHistory(self, item_type, source_key):
-        return self._makeGetRequest("/" + item_type + "/" + source_key + "/history")
+    def _get_item_history(self, item_type, source_key):
+        return self._make_get_request("/" + item_type + "/" + source_key + "/history")
 
-    def _getVersion(self, item_type, id):
-        return self._makeGetRequest("/versions/" + item_type + "/" + str(id))
+    def _get_version(self, item_type, id):
+        return self._make_get_request("/versions/" + item_type + "/" + str(id))
 
     '''
     EDGE METHODS
     '''
 
-    def createEdge(self, source_key, name, from_node_id, to_node_id, tags={}):
+    def create_edge(self, source_key, name, from_node_id, to_node_id, tags=None):
+        if tags is None:
+            tags = {}
         endpoint = "/edges"
         body = {"sourceKey": source_key, "name": name, "fromNodeId": from_node_id, "toNodeId": to_node_id}
 
         if tags:
             body["tags"] = tags
 
-        return self._makePostRequest(endpoint, body)
-
-    def createEdgeVersion(self,
-            edge_id,
-            to_node_version_start_id,
-            from_node_version_start_id,
-            to_node_version_end_id=-1,
-            from_node_version_end_id=-1,
-            reference=None,
-            reference_parameters={},
-            tags={},
-            structure_version_id=-1,
-            parent_ids=[]):
+        response = self._make_post_request(endpoint, body)
+        if response is None:
+            return None
+        else:
+            return model.core.edge.Edge(response)
+
+    def create_edge_version(self,
+                            edge_id,
+                            from_node_version_start_id,
+                            to_node_version_start_id,
+                            from_node_version_end_id=-1,
+                            to_node_version_end_id=-1,
+                            reference=None,
+                            reference_parameters=None,
+                            tags=None,
+                            structure_version_id=-1,
+                            parent_ids=None):
+
+        if reference_parameters is None:
```

Comment:
Why is this being done here? This should be done in `_get_rich_version_json`.

Reply:
Looks like in `_get_rich_version_json` the values are only added if they evaluate to `True` anyway, so we can just remove all of these as `None` and empty collections all evaluate to `False`